middle_end/flambda2/tests/tools/flexpect.ml

```diff
@@ -118,6 +118,11 @@ let run_mdflx_file filename : Outcome.t =
     Error
 
 let _ =
+  Symbol0.force_runtime4_symbols ();
+  if not Config.stack_allocation
```

Comment:
Maybe leave a CR to revisit?
(assuming that `Config.runtime5`
will be removed soon, while
`Config.stack_allocation` will stay)